nasl/charcnv.c

```diff
@@ -91,7 +91,7 @@ check_dos_char_slowly_ntlmssp (uint16 c)
 
   len1 = convert_string_ntlmssp (CH_UTF16LE, CH_DOS, &c, 2, buf, sizeof (buf),
                                  False);
-  if (len1 == 0)
+  if (len1 <= 0)
```

Comment:
`convert_string_ntlmssp` is defined as `size_t` which is unsigned?

Reply:
making len1 and len2 `size_t` var, ccc still complains.

Reply:
yes `convert_string_ntlmssp` is using `(size_t) -1` as an error code and casting to int will get -1 so I'm wrong and `<= 0` is correct.